File: my-folder/0242-valid-anagram/solution.ts
function isAnagram(s: string, t: string): boolean {

    if (s.length !== t.length) return false;

    let map = new Map();

    for (let c of s) {
        if (map.has(c)) {
            map.set(c, map.get(c) + 1);
        } else {
            map.set(c, 1);
        }
    }

    for (let c of t) {
        if (map.has(c) && map.get(c) > 0) {
            map.set(c, map.get(c) - 1);
        } else {
            return false;
        }
    }

    return true;
};

File: my-folder/0242-valid-anagram/solution.py
class Solution(object):
    def isAnagram(self, s, t):
        if len(s) != len(t):
            return False
        
        for c in set(s):
            if s.count(c) != t.count(c):
                return False
        return True
        
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        
